server/src/test/java/org/elasticsearch/cli/MultiCommandTests.java

```diff
@@ -22,22 +22,58 @@
 import joptsimple.OptionSet;
 import org.junit.Before;
 
+import java.io.IOException;
+import java.util.concurrent.atomic.AtomicBoolean;
+
 public class MultiCommandTests extends CommandTestCase {
 
     static class DummyMultiCommand extends MultiCommand {
+
+        final AtomicBoolean closed = new AtomicBoolean();
+
         DummyMultiCommand() {
-            super("A dummy multi command", () -> {});
+            super("A dummy multi command", () -> {
+            });
+        }
+
+        @Override
+        public void close() throws IOException {
+            super.close();
+            if (!this.closed.compareAndSet(false, true)) {
```

Comment:
Typically we use `== false` for negations (much easier to spot when glancing through some code).